### PR TITLE
SDK 3.45.0-preview.0: Adds Version bump and updates changelog

### PR DESCRIPTION
**Pull request by philipthomas-MSFT**
# Pull Request Template

### Public contract changes:

API_3 44 0

### Preview contract changes:

API_3 45 0-preview 0

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -15,7 +15,27 @@ Preview features are treated as a separate branch and will not be included in th
...
+
+#### Added
+
+- [4566](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4566) Container: Added support for IsFeedRangePartOfAsync, enabling precise comparisons to determine relationships between FeedRanges.
+
+### <a name="3.44.0"/> [3.44.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.44.0) - 2024-10-07
+
+#### Added
+
+- [4724](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4724) Query: Added support for deserializing HybridSearchQueryInfo in the query plan using Newtonsoft.Json.
```
Comment (kirankumarkolli) [inline changelog.md:28]: Is this meant for release notes (may be PR not qualified as internal can you please check with Neil)

Comment (philipthomas-MSFT) [reply changelog.md]: Talk to Neil, removing 4724.

**Review by NaluTripician — APPROVED** [review]
LGTM. Just and FYI for next time, the preview + normal version PRs can be combined :)

**Review by kundadebdatta — APPROVED** [review]
LGTM.

**Comment by kirankumarkolli** [comment]
One PR with preview or GA one enough right? 
Can you please update the release WIKI accordinlgy.

**Review by kirankumarkolli — APPROVED** [review]

**Comment by philipthomas-MSFT** [comment]
> One PR with preview or GA one enough right? Can you please update the release WIKI accordinlgy.

Closing the other one. And I will make some updates to documentation.

**Pull request merged**